detox/src/artifacts/templates/plugin/ArtifactPlugin.js

```diff
@@ -44,13 +44,13 @@ class ArtifactPlugin {
    *
    * @protected
    * @async
-   * @param {Object} event - Launch app event object
-   * @param {string} event.deviceId - Current deviceId
-   * @param {string} event.bundleId - Current bundleId
-   * @param {Object} event.launchArgs - Mutable key-value pairs of args before the launch
+   * @param {Object} _event - Launch app event object
+   * @param {string} _event.deviceId - Current deviceId
+   * @param {string} _event.bundleId - Current bundleId
+   * @param {Object} _event.launchArgs - Mutable key-value pairs of args before the launch
    * @return {Promise<void>} - when done
    */
-  async onBeforeLaunchApp(event) {
+  async onBeforeLaunchApp(_event) {
```

Comment:
I'm not very enthusiastic about this approach. It makes the argument's crippled name the official one, for the API. It makes it weird esp. in these cases where classes inherit these methods -- both in calling `super()`, and when called as an external API.